main/js/download-scritps.js

```diff
@@ -1,5 +1,67 @@
+// Installer: ARM, x64, x86 THEN Portable: ARM, x64, x86
+const windowsLinks = [
+  "https://github.com/KnossosNET/Knossos.NET/releases/download/v0.2.0-RC10/Knossos.NET-0.2.0-RC10-arm64.ex", 
```

Comment:
You'll probably replace this, but just in case it's missed, the final "e" got removed from this line somehow.